sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -196,6 +196,20 @@ def decode(self, encoded):
     return self._pickle_coder.decode(encoded)
 
 
+class ProtoCoderImpl(CoderImpl):
```

Comment:
You should actually be inheriting from SimpleCoderImpl so you get the encode/decode_from_string implementations for free.  Currently, your coder won't work if nested in, for example, a TupleCoder (also, please add a test for that case).